tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/management/NotificationTest.java

```diff
@@ -79,13 +79,17 @@ public void testBINDING_ADDED() throws Exception {
 
       NotificationTest.flush(notifConsumer);
 
+      long start = System.currentTimeMillis();
       session.createQueue(address, queue, durable);
 
       //the first message received will be for the address creation
       ClientMessage[] notifications = NotificationTest.consumeMessages(2, notifConsumer);
       Assert.assertEquals(BINDING_ADDED.toString(), notifications[1].getObjectProperty(ManagementHelper.HDR_NOTIFICATION_TYPE).toString());
       Assert.assertEquals(queue.toString(), notifications[1].getObjectProperty(ManagementHelper.HDR_ROUTING_NAME).toString());
       Assert.assertEquals(address.toString(), notifications[1].getObjectProperty(ManagementHelper.HDR_ADDRESS).toString());
+      Assert.assertTrue(notifications[1].getTimestamp() > start);
+      Assert.assertTrue((long) notifications[1].getObjectProperty(ManagementHelper.HDR_NOTIFICATION_TIMESTAMP) >= start);
```

Comment:
I would use Assert.assertThat and Matchers.greaterXXX so when it fails we have a more communicative error (that is just a lil' thing but I have introduced the core matchers of hamcrest for this reason some time ago)

Reply:
I'm trying to understand what this means?? :) can you clarify?

What for anyways? it's just an assertion? I guess I'm not understanding you.

Reply:
It just means that the notification was sent *after* the binding was added. It's a simple validation of the timestamp.